bundles/automation/org.eclipse.smarthome.automation.core/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,8 @@ Import-Package:
  org.eclipse.smarthome.core.storage,
  org.osgi.framework,
  org.osgi.service.cm,
+ org.osgi.service.component;version="1.2.2",
```

Comment:
https://www.eclipse.org/smarthome/documentation/development/guidelines.html
"The manifest must not have any version constraint on package imports, unless this is thoughtfully added. Note that Eclipse automatically adds these constraints based on the version in the target platform, which might be too high in many cases."

So, if you know that this bundle does not work using a previous version (e.g. "1.2.1") of the package, I am fine with the version constraint.
If the constraint has been added because the target version uses 1.2.2, you should remove that / replace that with the correct minimum version.